README.md

```diff
@@ -1 +1,35 @@
-# portfolio-1-microverse
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Portfolio
+
+> Description of Portfolio.
+This is the "Portfolio" Project with just Html and Css
+
+## Built With
+
+Html and Css
+
+
+## Getting Started
+
+npm clone https://github.com/heintayzar-hm/portfolio-1-microverse.git
+
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+You have basic knowledge of Html, Css
+### Setup
+
+### Install
+npm install
+### Usage
+
+### Run tests
+You can just go index.html file and run it.
+### Deployment
+
+## Contact me
+Github:https://github.com/heintayzar-hm/
+LinkedIn:https://www.linkedin.com/in/hein-tay-zar/
+Twitter:https://twitter.com/heintayzarhm
```

Comment:
- [OPTIONAL] I suggest adding the `MIT.md` file to your project and a Link to this file in your README 👌, Here is the [Micorverse Template](https://github.com/microverseinc/readme-template/blob/master/MIT.md), and remember to customize it 😉

> The MIT license gives users express permission to reuse code for any purpose, sometimes even if the code is part of proprietary software. As long as users include the original copy of the MIT license in their distribution, they can make any changes or modifications to the code to suit their own needs. 💯